test/functional/caps-e2e-specs.js

```diff
@@ -34,13 +34,16 @@ describe('Mac2Driver - caps', function () {
     }
   });
   beforeEach(async function () {
-    driver = wd.promiseChainRemote(HOST, PORT);
-    await driver.init(CAPS);
+    driver = await remote({
+      hostname: HOST,
+      port: PORT,
```

Comment:
using a static port _may_ be problematic; use [get-port](https://npm.im/get-pport) if possible.

Reply:
is there any useful example I can use?

Reply:
I think I merged a PR a while back that incorporated it into a test suite. Maybe a plugin? It seems like we'll want it to be a test convenience helper